tests/test_ahv.py

```diff
@@ -413,16 +413,6 @@ def test_validate_ahv_PE_config(self):
         result = self.ahv_config.validate()
         self.assertEqual(len(result), 0)
 
-    def test_validate_ahv_invalid_server_ip(self):
```

Comment:
Why is this test being removed? (Sorry it is not apparent to me for some reason)